tests/integration/it_employees_test.py

```diff
@@ -48,10 +47,21 @@ def test_timestamp_mixin_updated_on(db_session):
 
 
 def create_employee():
+    """ Create new instance of Employee to reuse in other tests """
     return Employee(
         first_name="John", last_name="Smith",
         username="john", phone_number="123", birth_date=datetime.utcnow(),
         registration_date=datetime.utcnow(), account_status="A",
         user_status="Working", email="test@test.com", password="bla",
         pin_code=1234
     )
+
+
+def test_list(client, db_session):
+    """ List all employees """
+    employee = create_employee()
+    db_session.add(employee)
+    db_session.commit()
+    response = client.get("/employees/")
```

Comment:
@bkuzmic One more point I didn't notice before. mb we will use `url_for` instead of hardcoded urls?

Reply:
@vryazanov Sure, no problem.